source/requirements.rst

```diff
@@ -0,0 +1,111 @@
+Design Requirements
+*******************
+
+The design requirements have been developed iteratively through testing with
+beamline scientists and users at NSLS-II. They will continue to change as we
+learn.
+
+Data Model
+==========
+
+* The event-based document model is the core of the project. It is not tied to
+  any specific language or storage technology.
+* Nothing in the schema will be specific to NSLS-II or synchrotron science. It
+  will be kept general to experimental science.
+* A facility or a beamline can impose extra required fields in the schema at
+  configuration time (e.g. requiring username, SAF number, proposal ID, or some
+  beamline-specific metadata).
+* At NSLS-II, there is yet no facility-wide standard for collecting information
+  identifying the owner of a data set. We await guidance from the user task
+  force on this issue. Ultimately, we expect that a set of required fields will
+  likely be imposed at the facility level, once the tooling is in place to
+  authenticate the user and validate their inputs.
+* At NSLS-II, there is no facility-wide standard for collecting sample
+  information. We expect that sample metadata fields will *never* be required at the
+  facility level (beyond the general SAF number) because some beamlines would
+  find such a requirement onerous and unhelpful. But a suggested set of prompts
```

Comment:
Playing a bit of defense here against a common line of questioning. Happy to remove this if it sticks out.